File: dxbase/src/main/java/com/dxmovie/dxbase/core/BaseConfig.java
package com.dxmovie.dxbase.core;




/**
 * @Description
 */
public class BaseConfig {
    public static int ENV = Environment.DEV;

    public static boolean isDEV() {
        return ENV == Environment.DEV;
    }

    public static boolean isProduct() {
        return ENV == Environment.PRODUCT;
    }

}

File: dxbase/src/main/java/com/dxmovie/dxbase/core/BaseConfig.kt
package com.dxmovie.dxbase.core

object BaseConfig {

    private var ENV = Environment.DEV

    @JvmStatic
    fun isDEV(): Boolean {
        return ENV == Environment.DEV
    }

    @JvmStatic
    fun isProduct(): Boolean {
        return ENV == Environment.PRODUCT
    }
}